src/filterConfig.js

```diff
@@ -243,6 +243,7 @@ const operators = [
 const booleanOperators = [
   { label: 'ui-inventory-es.and' },
   { label: 'ui-inventory-es.or' },
+  { label: 'ui-inventory-es.not' },
```

Comment:
Please do not use translation for the query formation
Instead add value and for the label use constant

Reply:
Thanks, fixed